mteb/evaluation/evaluators/RerankingEvaluator.py

```diff
@@ -251,3 +253,165 @@ def ap_score(is_relevant, pred_scores):
         # ap = np.mean([np.mean(preds[: k + 1]) for k in range(len(preds)) if preds[k]])
         ap = average_precision_score(is_relevant, pred_scores)
         return ap
+
+
+class MIRACLRerankingEvaluator(RerankingEvaluator):
+    def __init__(
+        self,
+        samples,
+        mrr_at_k: int = 10,
+        name: str = "",
+        similarity_fct=cos_sim,
+        batch_size: int = 512,
+        use_batched_encoding: bool = True,
+        limit: int = None,
+        k_values: List[int] = [1, 3, 5, 10, 20, 100, 1000],
```

Comment:
```suggestion
        k_values: list[int] = [1, 3, 5, 10, 20, 100, 1000],
```
as we have: `from __future__ import annotations`